src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -488,6 +488,68 @@ def check_if_recording_traces_fit_into_memory(recording: SpikeInterfaceRecording
         raise MemoryError(message)
 
 
+def iterize_recording_traces(
```

Comment:
I'd go with `iterate_recording_traces` as being a bit clearer

Reply:
I don't feel comfortable with "iterize" to be honest. 
On the other hand, "iterate" is used in Python for iterating over something so I don't think that overloading the semantics of this with "making something an iterator" is a good choice either.

What do you think of `build_recording_traces_iterator`, `wrap_recording_traces_on_iterator`?
Maybe I can think on something better.

Reply:
`setup_trace_iteration`?

Reply:
I decided to be more specific as I realized iterator is overly broad:

`_recording_traces_to_hdmf_iterator` 

I would prefer three words instead but as it is a private method I think is fine for it to focus on readability instead of type-ability.

Reply:
Sure thing